managed/src/main/java/com/yugabyte/yw/controllers/EncryptionAtRestController.java

```diff
@@ -44,13 +49,44 @@
 
 public class EncryptionAtRestController extends AuthenticatedController {
     public static final Logger LOG = LoggerFactory.getLogger(EncryptionAtRestController.class);
+    static final String SMARTKEY_KMS_PROVIDER = "SMARTKEY";
```

Comment:
Just use https://github.com/yugabyte/yugabyte-db/blob/master/managed/src/main/java/com/yugabyte/yw/common/kms/util/KeyProvider.java#L22 enums, there's no reason to re-define the different key provider types as strings here.